feeds/src/config.test.ts

```diff
@@ -0,0 +1,104 @@
+import { partialAsFull } from '@chainlink/ts-helpers'
+import { Config } from './config'
+
+describe('config', () => {
+  it('returns infura key from the process env', () => {
+    const processEnv = {
+      NODE_ENV: 'test' as const,
+      PUBLIC_URL: 'https://test.dev',
+      REACT_APP_INFURA_KEY: 'infuraKey',
+    }
+
+    expect(Config.infuraKey(processEnv)).toEqual('infuraKey')
```

Comment:
If you want, you could mutate `process.env` directly here so you don't have to have that default arg.

```
const originalEnv = JSON.parse(JSON.stringify(process.env))
beforeEach(() => { process.env = originalEnv })

...
process.env = processEnv
expect(Config.infuraKey()).toEqual('infuraKey')
....